app/src/org/commcare/connect/ConnectConstants.java

```diff
@@ -8,8 +8,15 @@
 public class ConnectConstants {
     public static final int ConnectIdTaskIdOffset = 1000;
 
-    public static int CREDENTIAL_PICKER_REQUEST = 2000;
+    public final  static int CREDENTIAL_PICKER_REQUEST = 2000;
     public static final String METHOD = "METHOD";
+    public static  String recoverPhone ;
```

Comment:
We only want constants (i.e. static final) in this class. These fields should move to ConnectIdActivity ideally or ConnectManager if not.